services/core-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -73,6 +75,37 @@ const documentColumns = [
   uploadDateColumn("upload_date"),
 ];
 
+const alertText = (status, updateUser, updateDate, responsibleInspector, selectedStatusCode) => {
+  let text = "";
+  if (selectedStatusCode === "UNR" && !responsibleInspector) {
+    text = `Please select an inspector responsible for this incident before changing the status to "Under Review".`;
+  } else if (status === "WNS") {
+    text = `This incident was submitted on ${formatDate(
+      updateDate
+    )} by ${updateUser} and has not yet been reviewed.`;
+  } else if (status === "UNR") {
+    text = `This incident currently under review by ${responsibleInspector}.`;
+  } else if (status === "RSS") {
+    text = `This incident's severity may have incorrectly been determined by the proponent, further clarification is being requested by the proponent.`;
+  } else if (status === "IMS") {
+    text = `This incident is missing critical information. The proponent has been notified that further information is required.`;
+  } else if (status === "AFR") {
+    text = `The incident was determined to be a dangerous occurence by the reporter and requires a final report to be submitted.`;
+  } else if (status === "INV") {
+    text = `This incident currently an EMLI under investigation.`;
+  } else if (status === "MIU") {
+    text = `This incident currently under MIU investigation.`;
+  } else if (status === "FRS") {
+    text = `This incident contains a final report submitted.`;
```

Comment:
Not sure this text makes sense.

Reply:
I agree. text is coming from the designs

Reply:
I would either re-word it or get it re-worded cause it is going to have to happen regardless